guides/installation/requirements.md

```diff
@@ -49,35 +51,122 @@ You can use these commands to check your actual environment:
   * `ext-xml`
   * `ext-zip`
   * `ext-zlib`
-* Composer recommended version: 2.0 or higher
+* Composer recommended version: 2.2 or higher
+
+<Tabs>
```

Comment:
This section looks a bit out of place IMHO, shouldn't we add an intro sentence like:

`This is how you install those requirements onto your machine?` or smth similar?